diff --git a/dist/index.js b/dist/index.js
index 914285807221289f4a9b822fbcf18bec8e6da588..0e4ca6e9f486142bb7e1715fdc84289e119a60e8 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,4 +1,5 @@
-import { getHighlighter } from 'shiki';
+// import { getHighlighter } from 'shiki';
+import { getSingletonHighlighter as getHighlighter } from 'shiki';
 import { visit } from 'unist-util-visit';
 import { toString } from 'hast-util-to-string';
 import rangeParser2 from 'parse-numeric-range';
